.coderabbit.yaml

```
@@ -0,0 +1,97 @@
+        - @DisplayName 문장형, 도메인 용어 사용.
+        - Given/When/Then BDD 스타일 유지.
+
+  finishing_touches:
+    docstrings:
+      enabled: true
+    unit_tests:
+      enabled: true
+
+  tools:
+    # 정적/보안 도구: 레포에 설정 파일이 있으면 더 강력해짐
+    pmd:          { enabled: true,  config_file: "" }
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)